config/vars.template.json

```diff
@@ -7,5 +7,6 @@
   "OFFICE_STATUS_CHANNEL_ID": "CHANNEL_ID",
   "RESUME_CHANNEL_ID": "CHANNEL_ID",
   "IRC_USER_ID": "USER_ID",
-  "MOD_USER_ID_FOR_BAN_APPEAL": "USER_ID"
+  "MOD_USER_ID_FOR_BAN_APPEAL": "USER_ID",
+  "CODEY_COIN_ROLE_ID": "ROLE_ID"
```

Comment:
Nitpick: Can this ROLE_ID be bundled together with the other ROLE_ID's towards the top? Same goes for `config/production/vars.json` and `config/staging/vars.json`.